js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +99,35 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating requestCustomPermissions function to MiniAppBridge object
+   * @param [string, string] permissionTypes, Type of custom permissions that is requested
+   * using an Array including the parameters eg. name, description.
+   *
+   * Miniapps should use "rakuten.miniapp.user." prefix for a valid name of the custom permissions
+   * For eg., Miniapps can pass the array of valid custom permissions as following
+   * [
+   *  {"name":"rakuten.miniapp.user.USER_NAME", "description": "Reason to request for the custom permission"},
+   *  {"name":"rakuten.miniapp.user.PROFILE_PHOTO", "description": "Reason to request for the custom permission"},
+   *  {"name":"rakuten.miniapp.user.CONTACT_LIST", "description": "Reason to request for the custom permission"}
+   * ]
+   *
+   * For eg., Miniapps can pass the array of invalid custom permissions as following
+   * [
+   *  {"name":"any unknown name", "description": "Reason to request for the custom permission"}
+   * ]
+   */
+  requestCustomPermissions(permissionTypes: [string, string]) {
```

Comment:
I think that `[string, string]` defines an Array that contains two string values, i.e. `['first string', 'second string']`. But this is supposed to be an array that contains objects, right? So I think it should be something like this:
```suggestion
  requestCustomPermissions(permissionTypes: Array<{name: string, description: string}>) {
```

Reply:
How about creating an object for this? `Array<CustomPermission>`?

Reply:
Yeah, making a `CustomPermission` type is even better.

Reply:
Thanks, I have updated.